projects/v2/src/app/modules/navbar/navbar.component.ts

```diff
@@ -101,12 +101,18 @@ export class NavbarComponent implements OnInit {
       this.selectedOrgans = organs;
       this.selectedOrgansValues =
       organs
-        ?.map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+        ?.map((s) => {
+          s = s.split('-')[0];
```

Comment:
This is a hack. Can you find a way to use the sheet config's organ name instead of hacky string manipulation?